### DISCUSSION TITLE
How much network bandwidth does Tshock require?

### DISCUSSION THREAD

**Discussion by TsXor** [Pryaxis/TShock#2553] [Q&A]
So I'm opening a server on a computer in my home with debian11, 4GB RAM, 4c4t 3GHz x4 640 (I think this is enough).
I don't have a public ip and what's worse, I'm in symmetric NAT. That's why I need to use something like frp or ngrok. Now I need advise on network bandwidth. How much will be enough to serve about 6 players?

**Comment by ACaiCat** [comment]
6 players use<3Mbps network bandwidth as usual,So 5Mbps is enough
